Core/PARStore.h

```diff
@@ -95,17 +96,41 @@ extern NSString *PARStoreDidSyncNotification;
 // This method returns an array of PARChange instances. It should not be called from within a transaction, or it will fail.
 - (NSArray *)fetchChangesSinceTimestamp:(nullable NSNumber *)timestamp;
 
+/// This method returns an array of PARChange instances for the device identifier passed in.
+/// It should not be called from within a transaction, or it will fail.
+/// Pass in nil for the device identifier to get results for all devices.
+- (NSArray *)fetchChangesSinceTimestamp:(nullable NSNumber *)timestampLimit forDeviceIdentifier:(nullable NSString *)deviceID;
+
+/// This method returns an array of PARChange instances for the device identifier passed in, between (including) the timestamps passed.
+/// It should not be called from within a transaction, or it will fail.
+/// Pass in nil for the device identifier to get results for all devices.
+/// Pass nil for either timestamp to have an open range.
+- (NSArray *)fetchChangesFromTimestamp:(nullable NSNumber *)firstTimestamp toTimestamp:(nullable NSNumber *)lastTimestamp forDeviceIdentifier:(nullable NSString *)deviceID;
```

Comment:
Yeah, I on purposely didn't comment on all the places where I could see the word `uuid` anywhere as I knew I would revisit those anyway when doing the big rename. I just highlighted the ones I would be afraid to miss otherwise. Outside the ones I mentioned don't bother about changing any in what you've written already and indeed just focus on the going forward.